frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -363,6 +366,40 @@ public int getNumberOfGpu() {
         return getIntProperty(TS_NUMBER_OF_GPU, 0);
     }
 
+    public String getMetricsConfig() {
+        // Getting absolute path of working directory, and finding the metrics_default yaml.
```

Comment:
Is this a safe assumption? I don't think we now expect people to git clone serve before they can run it

Reply:
The assumption was that users had to be running torchserve within the serve directory, which doesn't seem to always be the case if I'm interpreting your comments correctly. So it doesn't seem to be a safe assumption.

What I'm trying to do is if the `metrics_config` argument isn't specified by the user, then the default YAML file (which is found in frontend/server/src/test/resources/metrics_default.yaml) is used. I'm having trouble coming up with ways to pass that default yaml path. A relative import seems possible, but not sure how to exactly go about it

Reply:
So can users run torchserve outside of the `serve` repo / not need the `serve` repo to run it? If so, this method won't be very optimal. Is there a way to make a generalized link to the default yaml file in `frontend/server/src/test/resources/metrics_default.yaml` that can be accessed without using the `serve` repo?